blog/2019-09-ngrx-data-views/README.md

```diff
@@ -0,0 +1,419 @@
+---
+title: "Handling normalized entities with data views"
+author: Danilo Hoffmann
+mail: dhhyi@aol.com
+published: 2019-09-30
+last-change: 2019-09-30
+keywords:
+  - Angular
+  - RxJS
+  - Reactive Programming
+  - Observables
+  - NgRx
+  - NgRx Entities
+  - Memoization
+language: en
+thumbnail: data-views-header.jpg
+hidden: false
+---
+
+Dealing with data supplied by enterprise systems can be quite tedious. Most of the time, data for a single entity has to be retrieved with multiple calls, because it is handled in a normalized way on the server. This Normalization however does a good job minimizing the space used up in any database by preventing duplication and with that also inconsistency. Have a look at the [Wikipedia Article on Database Normalization](https://en.wikipedia.org/wiki/Database_normalization) for more information.
```

Comment:
> This Normalization however does a good job

Improve transition, e.g. "However, this is not bad since the normalization does a good job"

Reply:
I like that :grin: